src/types/test-failed/case-26.tact

```diff
@@ -0,0 +1,9 @@
+struct A {
+    x: Int;
+    y: Int;
+}
+
+const a: Int = 5;
+const b: Int = 6;
+
+const D: A = A { a, b };
```

Comment:
This should not fail: both `a` and `b` are in scope

Reply:
So, should we rely on ordering to assign the fields? I thought this should fail as the fields (valid identifiers) are `x` and `y`, not `a` and `b`.

ie.

```
struct A {
    x: Int;
    y: Int;
}

const a: Int = 5;
const b: Int = 6;
const x: Int = 7;
const y: Int = 8;

const D: A = A { a, b } /* should fail as there are no fields a and b */
const E: A = A { x, y } /* should resolve to A{ x: 7, y: 8 } */
```

What am I missing?

Reply:
ah, sorry, my bad -- too much code review on Sunday :)

The error message is not the right one, though. The error should be at the instance definition point, not the struct definition. You need to add the following to `case-26.tact`:

```
primitive Int;

trait BaseTrait {
    
}
```

Reply:
Hah, no worries at all! Thanks for taking the time to review it on a Sunday. I usually try do something different from the day job and playing with Tact and TON has been a favourite lately.

Reply:
@vitorpy Nice to hear that! Looking forward to have more contributions from you :)